contracts/src/bridge/frontend/Dockerfile

```diff
@@ -1,7 +1,9 @@
-# Use an official Node.js 22 as a parent image
-FROM node:22-alpine
+FROM node:18-buster as runner
```

Comment:
Probably better to use newest version if it works (like changing `node:18-buster` -> `node:22-alpine`). Shouldn't make a difference but might avoid some issues.